src/uproot/_dask.py

```diff
@@ -886,7 +915,11 @@ def _get_dak_array(
     is_self = []
 
     count = 0
-    for file_path, object_path in files:
+    for file_object_maybechunks in files:
+        file_path, object_path = file_object_maybechunks[0:2]
+        if len(file_object_maybechunks) == 3:
+            file_object_maybechunks[2]
```

Comment:
This doesn't do anything: the `file_object_maybechunks[2]` is accessed but not assigned. You must need this information further down. Let me see if you get it another way...

Reply:
checking - this may have been a leftover from previous development.

Reply:
Yes, indeed that should be:
```python3
chunks = file_object_maybechunks[2]
```

to assign the chunks - I may have added that in to the wrong function first by mistake and forgot to delete it.

You'll see the correct implementation further down.

Reply:
Got it. Then I'll delete this reference without assignment, since the active part is in the `_get_*_delay_open` functions.